tests/foreman/cli/test_repository.py

```diff
@@ -2566,5 +2565,21 @@ def test_positive_symlinks_sync(self):
         :expectedresults: entire directory is synced, including files
             referred by symlinks
 
-        :CaseAutomation: notautomated
+        :CaseAutomation: automated
```

Comment:
could you also change a tier for this test? i don't think tier1 is appropriate here. I would use tier2.